src/napari/layers/shapes/_accelerated_triangulate_python.py

```diff
@@ -482,6 +482,60 @@ def orientation(p: np.ndarray, q: np.ndarray, r: np.ndarray) -> np.ndarray:
     return val
 
 
+def vectorized_orientation(
```

Comment:
Expand up to the original scalar `orientation`. It's not used anywhere after the refactor of this PR. But for now I left it, because maybe it could be useful -- it was short so I left it as is.